concepts.md

```diff
@@ -0,0 +1,35 @@
+# Concepts
+
+## User
+
+Person identified with [WebID](https://www.w3.org/2005/Incubator/webid/spec/identity/).
+Assumed as trusted unless explicitly called **malicious user**.
+
+## WebID Profile
+
+Document resulting from dereferencing WebID [as specified](https://www.w3.org/2005/Incubator/webid/spec/identity/).
+No assumptions made about where it stays hosted.
+
+## Resource Server (RS)
+
+Server hosting resources protected with [Web Access Control](https://github.com/solid/web-access-control-spec).
+
+## Resource Owner
```

Comment:
note that in https://github.com/solid/web-access-control-spec#not-supported-by-design 

> WAC has no formal notion of a resource owner, and no explicit Owner access mode. For convenience/UI purposes, it may be assumed that Owners are agents that have Read, Write and Control permissions.

i personally think of a "Resource Owner" as being outside of what's represented in WAC, and might involve a business relationship with a resource server provider, or low-level out-of-band control of the resources (for example, with direct access to the storage filesystem with a Unix shell), that could grant the "owner" with administrative access even without `acl:Control` permission.  or in other words, "a controller could remove their own control permission, but an owner could get it back".

i would like to propose formalizing this distinction, particularly since practical Pod providers will need a way for a Pod's "owner" to recover from ACL mistakes (which might not necessarily be by API and LDP manipulation of ACL resources).  people today have locked themselves out of resources with NSS.

Reply:
> For convenience/UI purposes, it may be assumed that Owners are agents that have Read, Write and Control permissions.

Having `acl:Control` one can always add other permissions for oneself, on the other hand having just the other permissions one can't add `acl:Control`, in practice it seems to boil down to that specific access mode.

> i personally think of a "Resource Owner" as being outside of what's represented in WAC, and might involve a business relationship with a resource server provider.

Solid team has resources on `github.com`, some members have have *owner* access to them. I think we can consider them resource owners but it doesn't require any special relation to *GitHub, Inc*. Actually thinking of github, I may have only *read* permission and grant other clients access to those resources using OAuth2, in that case I would have *Resource Owner* role even though I only have *read* permission to that resource.
Maybe we shouldn't use this term all together and just use more verbose and precise constructs like *User with control access*, *User with any access* etc.

> i would like to propose formalizing this distinction, particularly since practical Pod providers will need a way for a Pod's "owner" to recover from ACL mistakes (which might not necessarily be by API and LDP manipulation of ACL resources)

How about simply Resource Server admin? Keeping in mind that if someone offers it as service, they may have no Web ACL access to any of the resources.